Game/Source/GDKTestGyms/FunctionalTests/TestActors/GASTestPawnBase.h

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "Abilities/GameplayAbility.h"
+#include "AbilitySystemComponent.h"
+#include "AbilitySystemInterface.h"
+#include "CoreMinimal.h"
+#include "GameFramework/Pawn.h"
+#include "GASTestPawnBase.generated.h"
+
+/**
+ * A replicated Actor with a Cube Mesh, used as a base for Actors used in spatial tests.
+ */
+UCLASS()
+class AGASTestPawnBase : public APawn, public IAbilitySystemInterface
```

Comment:
There's a fair bit of code duplication between this and the GASCharacter class that already exists in the test gyms project - the only difference is that this class has the automaticlaly granted abilities. 
This class could feasibly be a subclass of the existing one, but I'm not sure if we want to mix gyms and test classes.